Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/volume/empty_dir/empty_dir_linux.go

```diff
@@ -51,3 +52,8 @@ func (m *realMountDetector) GetMountMedium(path string) (storageMedium, bool, er
 	}
 	return mediumUnknown, isMnt, nil
 }
+
+// selinuxEnabled determines whether SELinux is enabled.
+func selinuxEnabled() bool {
```

Comment:
Will this cross compile?  Assuming it does because e2e passes, want to be sure).